修复PyCharm中， Blender print无法实时输出的问题

0MirageTank0 opened the pull request:
- 添加 --python-use-system-env 参数，使用与test.py相同的 PYTHONUNBUFFERED=1 环境变量,强制 Blender 运行在无缓冲模式 
- 修复若存在多个版本 无法正确加载插件的问题（Steam更新Blender时会保存旧版本的目录）
![image](https://github.com/user-attachments/assets/7fadef6d-aa17-436a-9720-6c9315b6fc48)

0MirageTank0 committed:
- Fix the issue where the PyCharm console does not respond in real-time to print
- Fix the issue where plugins cannot be loaded correctly if an old version exists

0MirageTank0 committed:
- 修复 Blender print无法实时输出的问题：在 subprocess.call 中添加 --python-use-system-env 参数，使用test.py相同的 PYTHONUNBUFFERED=1 环境变量,强制 Blender 运行在无缓冲模式
 - 修复若存在旧版本 无法正确加载插件的问题

xzhuah commented on main.py:
```diff
@@ -190,7 +190,7 @@ def exit_handler():
 
     try:
-        subprocess.call([BLENDER_EXE_PATH, "--python-expr", python_script])
+        subprocess.call([BLENDER_EXE_PATH, "--python-use-system-env", "--python-expr", python_script])
```
Please also add "--python-use-system-env" to line 169

xzhuah commented on common/class_loader/module_installer.py:
```diff
@@ -32,7 +32,9 @@ def install_if_missing(package):
 def extract_blender_version(blender_exe_path: str):
     folder_path = os.path.dirname(blender_exe_path)
-    for version in ["2.93", "3.0", "3.1", "3.2", "3.3", "3.4", "3.5", "3.6", "4.0", "4.1", "4.2"]:
+    version_list = ["2.93", "3.0", "3.1", "3.2", "3.3", "3.4", "3.5", "3.6", "4.0", "4.1", "4.2"]
```
let's just inline it in reverse order directly
["4.2", "4.1", "4.0", "3.6", "3.5", "3.4", "3.3", "3.2", "3.1", "3.0", "2.93"]

0MirageTank0 committed:
Merge remote-tracking branch 'origin/master'

0MirageTank0 committed:
- Fix the issue where the PyCharm console does not respond in real-time to print
- Fix the issue where plugins cannot be loaded correctly if an old version exists

0MirageTank0 committed:
Merge remote-tracking branch 'origin/master'
